React/my-app/src/service/accounts.js

```diff
@@ -4,10 +4,24 @@ export const postAccount = (url, body) => {
   return api.post(url, body);
 };
 
+export const getAccount = url => {
```

Comment:
Unnecessary function 😄

Reply:
Either call it getCurrentAccount and use '/user/me' instad of receiving an argument or remove it.